lib/Client.js

```diff
@@ -339,6 +343,60 @@ class Client extends EventEmitter {
         });
     }
 
+    /**
+    * Edits command permissions for a multiple commands in a guild.
+    * Note: You can only add up to 10 permission overwrites for a command.
+    * @arg {String} guildID The guild id
+    * @arg {Array<Object>} permissions An array of [partial guild command permissions](https://discord.com/developers/docs/interactions/application-commands#application-command-permissions-object-guild-application-command-permissions-structure)
+    * @returns {Promise<Array<Object>>} Returns an array of [GuildApplicationCommandPermissions](https://discord.com/developers/docs/interactions/application-commands#application-command-permissions-object-guild-application-command-permissions-structure) objects.
+    */
+    bulkEditCommandPermissions(guildID, permissions) {
+        if(!guildID) {
+            throw new Error("You must provide an id of the guild whose permissions you want to edit.");
+        }
+
+        return this.requestHandler.request("PUT", Endpoints.GUILD_COMMAND_PERMISSIONS(this.application.id, guildID), true, {permissions});
```

Comment:
This doesn't seem to work as Discord expects an [array of partial permissions](https://discord.com/developers/docs/interactions/application-commands#batch-edit-application-command-permissions) for bulk edits
Removing the curly brackets around `permissions` should fix it